### DISCUSSION TITLE
doc.ents does not recognize small letter place name

### DISCUSSION THREAD

**Discussion by ShojibDE** [explosion/spaCy#11638] [Help: Coding & Implementations] [answered]
I have trid with this code , look spacy model did not recognize the 'denver' and 'boston' as a Name entity. 

![spacy problem](https://user-images.githubusercontent.com/81525107/195417471-87cf6fe9-9f51-4231-91fc-6dbe651676b3.png)
import spacy
nlp = spacy.load("en_core_web_md")
doc= nlp("show me flights from denver to boston on Tuesday")
doc.ents

**Comment by thomashacker** [comment]
Hello,
I've tried to recreate your result with the same code, but for me, it was able to recognize all entities correctly. 
Which spaCy version are you using? You can see it using `spacy info` in the command line.

Generally, machine learning models are mostly not 100% accurate, and thus have some cases in which the predictions are not correct (see https://github.com/explosion/spaCy/issues/3052).

**Reply by ShojibDE** [reply]
Hi  thomashacker,
I am using spaCy version    3.0.8
![vSpacyproblem](https://user-images.githubusercontent.com/81525107/195641907-ef3d8af3-20f2-4852-b854-86a52cef84e0.png)

**Comment by adrianeboyd** [comment, marked as answer]
This is a known issue with lowercase augmentation for models with version v3.0.0. Please upgrade to a newer version of spacy and download newer models.

**Comment by ShojibDE** [comment]
Thanks a lot, adrianeboyd

The problem is now solved. But unfortunately, My thesis supervisor already rejected my thesis for this issue
![Solved](https://user-images.githubusercontent.com/81525107/196053592-07e131d5-63c8-444c-9d33-13e7c92daeb4.png)
.
